ElevenLabs-DotNet-Tests/TestFixture_03_VoicesEndpoint.cs

```diff
@@ -248,5 +250,60 @@ public async Task Test_12_01_IterateDefaultVoices()
                 Console.WriteLine($"{voice.Id} | {voice.Name}");
             }
         }
+
+        [Test]
+        public void Test_13_V2_TestEnumSerializtion()
```

Comment:
lets move this test into its own fixture, I think there may be one already for generic/extension stuff.